core/backend-itwin-client/src/imodelhub/AzureFileHandler.ts

```diff
@@ -185,9 +185,18 @@ export class AzureFileHandler implements FileHandler {
     }
   }
 
-  private async downloadFileUsingHttps(requestContext: AuthorizedClientRequestContext, downloadUrl: string, downloadToPathname: string, fileSize?: number, progressCallback?: ProgressCallback, cancelRequest?: CancelRequest): Promise<void> {
-    const bufferThreshold = (this.useBufferedDownload(downloadToPathname)) ? this._threshold : undefined;
-    return downloadFileAtomic(requestContext, downloadUrl, downloadToPathname, fileSize, progressCallback, cancelRequest, bufferThreshold);
+  private async downloadFileUsingHttps(_requestContext: AuthorizedClientRequestContext, downloadUrl: string, downloadToPathname: string, _fileSize?: number, progressCallback?: ProgressCallback, cancelRequest?: CancelRequest): Promise<void> {
+    let lastProgressStat: any;
```

Comment:
If AzCopy is configured that it will still be limited to small files. My plane was not to suddenly remove AzCopy instead do it in two steps. In first push this PR and let DR and GP backend to remove AzCopy config and run without it. If everything is stable and they do not find any issue. I will remove AzCopy altogether in next release.

I have not worked on uploader but it should be very similar where we can initiate upload of multiple block at same time. I will create Backlog item for it.